docs/vNext/Ethereum_Configuration.md

```diff
@@ -9,7 +9,7 @@ permalink: /vNext/ethereum-config/
 
 This page introduces the Ethereum adapter suitable for all the Ethereum clients that expose the web3 RPC interface over HTTP.
 
-> Geth is currently the only tested client but, under fully web3 compliance, there would not be any reason to not work with other clients.
+> Hyperledger Besu and Geth are the current tested clients. The tests are driven via standard Ethereum JSON-RPC APIs so other clients should be compatible once docker configurations exist.
```

Comment:
I would add here, that Besu requires the *privateKey options set

Reply:
Done